setup.py

```diff
@@ -55,6 +55,12 @@ def get_install_requires():
     include_package_data=True,
     zip_safe=False,
     install_requires=get_install_requires(),
+    extras_require={
+        'notifications': [
+            # TODO: uncomment when openwisp-notifications is released
+            # 'openwisp_notifications~=0.1',
```

Comment:
I think we'll have to put it in the default requirements.

Reply:
In the project idea, it was stated to make it an optional dependency. Isn't this the way to go? 

From http://openwisp.org/gsoc/ideas-2020.html:
> Make the notifications module an optional dependency of openwisp-controller (by creating a sub app in openwisp-controller) and create the following notification types in it:

Reply:
It was conservative. I think we need this functionality to make the user more aware of what goes on in the system and we hence it makes sense to make it a requirement. Do you see any problem with this?